src/components/admonition/admonition.scss

```diff
@@ -0,0 +1,41 @@
+.admonition {
+    display: flex;
+    width: 100%;
+
+    &.danger {
+        background-color: rgb(var(--color-red-dark));
+    }
+    &.info {
+        background-color: rgb(var(--color-cyan-default));
+    }
+    &.note {
+        background-color: rgb(var(--color-gray-default));
+    }
+    &.tip {
+        background-color: rgb(var(--color-green-dark));
+    }
+    &.warning {
+        background-color: rgb(var(--color-orange-dark));
+    }
+
+    .admonition-side {
+        mix-blend-mode: screen;
+        background-color: rgb(var(--contrast-800));
+        padding: 0.25rem;
+    }
+    .admonition-main {
+        mix-blend-mode: screen;
+        background-color: rgb(var(--contrast-300));
+        padding-left: 0.5rem;
+        width: 100%;
+
+        .admonition-main-title {
+            margin: 0.1rem;
+            margin-top: 0.25rem;
+        }
+        .admonition-main-description {
+            margin: 0.1rem;
+            white-space: pre-line;
```

Comment:
Could we support markdown here, instead of just supporting linebreaks? 🤔

Reply:
I also would prefer markdown support. Also due to this, I'd say that this `<p class="description">` shouldn't be a `<p` element. It should be just a span or something.

Is it hard to add markdown support?

Reply:
There are too many cooks in this soup at the moment, so I'm withdrawing.